.github/workflows/do_release.yml

```
@@ -0,0 +1,23 @@
+name: Create a GitHub release once a release PR is merged
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release')
```

```suggestion
    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release') && github.repository_owner == 'packit'
```

just making sure it's consistent with ogr